lxd/storage/backend_lxd.go

```diff
@@ -3988,7 +3994,9 @@ func (b *lxdBackend) ImportCustomVolume(projectName string, poolVol *backup.Conf
 			return err
 		}
 
-		revert.Add(func() { revert.Add(func() { VolumeDBDelete(b, projectName, fullSnapName, drivers.VolumeTypeCustom) }) })
+		revert.Add(func() {
+			revert.Add(func() { _ = VolumeDBDelete(b, projectName, fullSnapName, drivers.VolumeTypeCustom) })
```

Comment:
This looks odd. Is this correct?

Reply:
Hrm yeah that is certainly a bug.

Reply:
Fixed this in https://github.com/lxc/lxd/pull/10427/commits/a7320e98415509b18b7e3733803923338522fcb1